README.md

```diff
@@ -20,10 +33,34 @@
     }
   }
   ```
-- Launch loki, grafana, jaeger, vector, opentelemetry-collector
-  ```
+  > Note: Instead of configuring the driver to be the Docker default it can also be setup per container using `docker run --log-driver=loki` and `--log-opt loki-url="..."`, or by adding the following to each container in your docker-compose.yaml file:
+  > ```yaml
+  > logging:
+  >   driver: loki
+  >   options:
+  >     loki-url: "..."
+  > ```
+- Launch cAdvisor, Grafana, Jaeger, Loki, OpenTelemetry Collector, Prometheus, and Vector:
+  ```sh
   docker-compose up -d
   ```
+- Navigate to <http://localhost:3000/> and log in using the default Grafana user (admin/admin).
+
+## Troubleshooting
+
+### Directory permissions
+
+If you see any errors about being unable to create files or directories when starting the containers then the issue is probably related to user permissions. We have chosen to run the containers which write to disk with uid 1000 and gid 1000. We see this as being a sensible default for most users, but if it causes problems you can either change the `user: "1000:1000"` lines in docker-compose.yaml to values which suit your environment, or remove the lines and set the correct permissions on the directories:
+```sh
+chown -R 472:472 ./data/grafana
+chown -R 0:0 ./data/jaeger
+chown -R 10001:10001 ./data/loki
```

Comment:
Is that still correct now you've added a `user: 1000:1000`. mapping?

Reply:
The intention is to say run that if you're having problems and you aren't logged in as user 1000 (after you've removed the `user: "1000:1000"` lines).

Reply:
I should have read the description above. 🤦 
That's good for me. Tested it with my existing directories and everything is still running.